watertap/unit_models/MD/membrane_distillation_base.py

```diff
@@ -696,7 +696,7 @@ def _get_performance_contents(self, time_point=0):
         if hasattr(self, "width"):
             var_dict["Membrane Width"] = self.width
 
-        expr_dict["Average Solute Flux (LMH)"] = self.flux_mass_avg[time_point] * 1000
+        expr_dict["Average Solute Flux"] = self.flux_mass_avg[time_point] * 1000
```

Comment:
@zacharybinger Was the conclusion that the multiplication of the pyomo variables with units by a float actually doesn't do anything in the printing, or these still correctly print the value in the appropriate units?